scripts/build-json/build-json.js

```diff
@@ -18,22 +17,26 @@ function walk(directory, filepaths) {
     }
 }
 
-function collectItems(directory) {
+function collectItems(directory, searchPaths) {
     let filepaths = [];
     walk(directory, filepaths);
-    filepaths = filepaths.map( filepath => filepath.slice(path.join(process.cwd(), './content/').length) )
+    filepaths = filepaths.map( 
+        filepath => filepath.slice(path.join(process.cwd(), './content/').length) 
+    ).filter(filepath => !searchPaths.length || searchPaths.some(searchPath => filepath.includes(searchPath)))
```

Comment:
To me, this line is really hard to read.

Reply:
This'll get fixed once we let Prettier tidy it up.